Dockerfile

```diff
@@ -9,21 +9,24 @@ RUN apt-get update && apt-get install -y \
 
 ARG CEPH_REPO_URL=https://download.ceph.com/debian-nautilus/
 RUN wget -q -O- 'https://download.ceph.com/keys/release.asc' | apt-key add -
-RUN apt-add-repository "deb ${CEPH_REPO_URL} xenial main"
+RUN true && \
+  apt-add-repository "deb ${CEPH_REPO_URL} xenial main" && \
+  apt-get update && \
+  apt-get install -y ceph libcephfs-dev librados-dev librbd-dev curl gcc g++
```

Comment:
While reformatting the multiple RUN commands into one, it is recommended to add a `true` command on the last line as well as the 1st line. I don't expect that we need to add more commands later on, so this is not a blocker here.